src/Microsoft.AspNetCore.Routing.Abstractions/RouteValueDictionary.cs

```diff
@@ -218,58 +212,46 @@ public void Add(string key, object value)
         {
             if (key == null)
             {
-                throw new ArgumentNullException(nameof(key));
+                throw new ArgumentNullException("key");
```

Comment:
Do we not like nameof anymore? I'm curious

Reply:
I never liked it to begin with. 

I eneded up copy-pasting this code somwehre where and then I had to get rid of nameof. I'll fix it.